app/meteor-accounts-saml/server/saml_utils.js

```diff
@@ -277,6 +332,41 @@ SAML.prototype.validateSignature = function(xml, cert) {
 	return sig.checkSignature(xml);
 };
 
+SAML.prototype.validateLogoutRequest = function(samlRequest, callback) {
+	const compressedSAMLRequest = new Buffer(samlRequest, 'base64');
+	zlib.inflateRaw(compressedSAMLRequest, function(err, decoded) {
+		if (err) {
+			debugLog(`Error while inflating. ${ err }`);
```

Comment:
this whole function could be rewritten to look more slim.. like returning in this case, so don't need to have an `else` block.

the same applies to all other conditions below

Reply:
tried to simplify a little.  Hopefully a bit better